docs/tutorials/scan-a-product-tutorial.md

```diff
@@ -0,0 +1,79 @@
+# Scan A Product To Get Nutriscore
+
+This is a basic tutorial that shows you how to scan a product to get the nutriscore field. 
+
+Meet Dave. Dave is an active Open Food Facts contributor and a developer who wants to build HealthyFoodChoices, an Android app aimed at conscious consumers that buy healthy products. He has a consumer called Anna. Anna wants to know more on the nutritional facts of Nutella - Ferrero from the HealthyFoodChoices app. Dave needs his app to make an API call to provide her with this information.
+
+<!-- I am not sure If I should retain this scenario of Dave and Anna, Pierre suggested it -->
```

Comment:
I would remove it, it does not bring any value, and sounds "fake".

Maybe we can replace it with something more factual:

This is a basic tutorial that shows you can get the Nutri-Score of a product, for instance to display it in a mobile app after scanning the product barcode.

(we don't really show how to scan a product, so I rephrased a bit)